src/components/common/SelectTypes.tsx

```diff
@@ -0,0 +1,85 @@
+import { Stack } from '@mui/material';
+import Checkbox from '@mui/material/Checkbox';
+import FormControl from '@mui/material/FormControl';
+import InputLabel from '@mui/material/InputLabel';
+import ListItemText from '@mui/material/ListItemText';
+import MenuItem from '@mui/material/MenuItem';
+import OutlinedInput from '@mui/material/OutlinedInput';
+import Select, { SelectChangeEvent } from '@mui/material/Select';
+
+import { ItemType } from '@graasp/sdk';
+import { ItemIcon } from '@graasp/ui';
+
+import { useBuilderTranslation, useEnumsTranslation } from '@/config/i18n';
+import { BUILDER } from '@/langs/constants';
+
+import { useFilterItemsContext } from '../context/FilterItemsContext';
+
+const ITEM_HEIGHT = 48;
+const ITEM_PADDING_TOP = 8;
+const MenuProps = {
+  PaperProps: {
+    style: {
+      maxHeight: ITEM_HEIGHT * 4.5 + ITEM_PADDING_TOP,
+    },
+  },
+};
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const { S3_FILE, ...BUILDER_ITEM_TYPES } = ItemType; // exclude S3 file
```

Comment:
Curious why ? If the issue is that LocalFile and S3File both represent a file, I would argue that you should select s3File preferably. Maybe we need to change the translation label for it to be less "technical".